src/lib.rs

```diff
@@ -278,6 +278,14 @@ pub struct FailedSwapInfo {
     pub created_at: SystemTime,
 }
 
+/// Information about a failed swap refund
+pub struct FailedSwapRefundInfo {
```

Comment:
Maybe rename this to `RefundFailedSwapInfo`, otherwise it's a bit confusing if a swap refund has failed or if a swap has failed and this is about refunding.

Reply:
Addressed in [b7788fc](https://github.com/getlipa/lipa-lightning-lib/pull/712/commits/b7788fc89a1140e66119e0c281e8f24ba8eb83c0)